build/makelib/golang.mk

```diff
@@ -162,6 +162,10 @@ go.test: go.vet go.fmt
 #	@$(GOHOST) test -v -i $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 	@$(GOHOST) test -cover $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 
+.PHONY: go.lint
+go.lint: $(GO_TOOLS_DIR)/golint
```

Comment:
this needs to be `go.lint: $(GO_TOOLS_DIR)/lint/$(GOHOSTOS)-$(GOHOSTARCH)/golint`